src/noteServices/trashService.js

```diff
@@ -0,0 +1,69 @@
+import axios from "axios"
+
+const AddNoteToTrash = async (data, _id) => {
+    const authToken = localStorage.getItem("authToken")
+    try {
+        const res = await axios.post(`api/notes/trash/${_id}`, {
+            note: data
+        },
+            {
+                headers: { authorization: authToken }
+            })
+        console.log(res)
+        return res;
+    }
+    catch (error) {
+        console.log("Error occured", error)
+    }
+}
+
+const RestoreFromTrash = async (data, _id) => {
+    const authToken = localStorage.getItem("authToken")
+    try {
+        const res = await axios.post(`/api/trash/restore/${_id}`, {
+            note: data
+        },
+            {
+                headers: { authorization: authToken }
+            })
+        console.log(res)
+        return res;
+    }
+    catch (error) {
+        console.log("error occured", error)
+    }
+}
+
+const ArchiveToTrash = async (data, _id) => {
+    const authToken = localStorage.getItem("authToken")
+    try {
+        const res = await axios.post(`/api/archives/trash/${_id}`, {
+            note: data
+        },
+            {
+                headers: { authorization: authToken }
+            })
+        console.log(res)
+        return res;
+    }
+    catch (error) {
+        console.log("error occured", error)
```

Comment:
use toast to show error and success messages

Reply:
Yes I will do it later on